src/components/Groups/Group/index.js

```diff
@@ -63,9 +67,35 @@ const Group = props => {
     getActiveMembers()
   }, [members, profiles, setActiveMembers, teams])
 
+  useEffect(() => {
+    // Get channel posts
+    const fetchPosts = async () => {
```

Comment:
Posts are already available in this component via props, so I'm wondering if there is extra gains using getPosts here. Posts are updated via websocket, so those should be always up-to-date anyways.

Reply:
See above. I might have done something wrong but couldn't manage to get this to work via props.